### PR TITLE
Mission and goal section

### PR DESCRIPTION
**Pull request by SafeEHA**
Created a section for the goals and missions of Metal3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,9 +28,40 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">Why Metal<sup>3</sup></h2>
...
+        automation provided by Kubernetes.
+      </p><br>
+      <p>
+        The goal of Metal<sup>3</sup> is to offer a unified, vendor-agnostic solution for 
+        deploying, managing, and scaling bare-metal infrastructure ensuring seamless 
+        integration with Kubernetes.
+      </p>
      </div>
      </section>
+     <!-- <section class="mk-why-baremetal mk-main__section">
```
Comment (elfosardo) [inline index.html:45]: remove all the commented out section

File: index.html [hunk 2]
```diff
@@ -28,9 +28,40 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">Why Metal<sup>3</sup></h2>
          backend for Machine objects from the Cluster API. <br> These components integrate
          seamlessly to leverage the Kubernetes ecosystem and automate the provisioning
          and management of bare-metal infrastructure.
-       </p>
+       </p><br>
+       <p>
+        Metal<sup>3</sup> mission is to empower organizations with a flexible,
```
Comment (elfosardo) [inline index.html:33]: let's move this up as first statement
also use "The Metal3 project" instead of just Metal3

File: _sass/_type.scss [hunk 3]
```diff
@@ -6,6 +6,10 @@
         border-left: var(--mk--BorderWidth--lg) solid var(--mk--BorderColor--100);
         padding-left: var(--mk--spacer--sm);
     }
+   &.mk-border {
+    border-bottom: var(--mk--BorderWidth--md) solid var(--mk--BorderColor--100);
+    // padding-left: var(--mk--spacer--sm);
```
Comment (elfosardo) [inline _sass/_type.scss:11]: let's remove this commented out code

**Comment by SafeEHA** [comment]
Closes #330

**Pull request merged**